### PR TITLE
mapping by name subcategory

### PR DESCRIPTION
**Pull request by neoed**

### CODE REVIEW

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryDTO.java [hunk 1]
```diff
@@ -24,4 +24,5 @@ public String getDescription() {
     public UUID getSubCategoryUUID() {
         return subCategoryUUID;
     }
+
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryDTO.java:27]: empty line - remove it please

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryController.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package com.inqoo.trainingservice.app.subcategory;
 
+import org.springframework.stereotype.Controller;
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryController.java:3]: unused, remove it please

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryService.java [hunk 3]
```diff
@@ -60,4 +60,8 @@ private boolean validateCharacters(String description) {
         return description.length() < limitDescription;
     }
 
+    List<String> getAllSubcategoryName() {
+        return subcategoryRepository.getAllSubcategoryName();
+    }
+
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryService.java:66]: empty line

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryDTO.java (hunk 1 above)
Comment (neoed) [reply src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryDTO.java:27]: done

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryController.java (hunk 2 above)
Comment (neoed) [reply src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryController.java:3]: Done

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryService.java (hunk 3 above)
Comment (neoed) [reply src/main/java/com/inqoo/trainingservice/app/subcategory/SubcategoryService.java:66]: Done

**Review by adamkulawik — APPROVED** [review]

**Pull request merged**